detection-rules/suspicious_request_for_quote_or_purchase.yml

```diff
@@ -0,0 +1,58 @@
+name: "Request for Quote or Purchase (RFQ|RFP) from suspicious sender"
+description: |
+  RFQ/RFP scams involve fraudulent emails posing as legitimate requests for quotations or purchases, often sent by scammers impersonating reputable organizations.
+  These scams aim to deceive recipients into providing sensitive information or conducting unauthorized transactions, often leading to financial loss, or data leakage.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound 
+  and 1 of (
+        (
+            (
+                length(recipients.to) == 0 or 
+                all(recipients.to, .display_name == "Undisclosed recipients")
+            )
+            and length(recipients.cc) == 0
+            and length(recipients.bcc) == 0
+        ),
+        (
+            sender.email.domain.root_domain in $free_email_providers
+            and any(headers.reply_to, .email.email != sender.email.email)
+            and any(headers.reply_to, .email.email not in $recipient_emails)
+        )
+  )
```

Comment:
this snippet is 🔥  just heads up that this doesn't mean there's necessarily a suspicious sender (in the rule name)

Reply:
Love the name change ty ty ty!